### ISSUE TITLE
Controls needed to work with MVS display

### ISSUE THREAD

**Issue by r12a** [w3c/mlreq#29]
This is an issue originally raised by Nasun-urt and Huqitu in the mlreq draft. Moving the issue here for discussion and elaboration before contacting the CSS WG.

> Whether the Mongolian vowel space mark MVS (0x180E): is displayed: CSS-MONGOLIAN-MVS:Display|Normal, when the option Display is designated, there will be a small hollow rectangular on the screen with the size in the font library, but the rectangular will not be shown when it is printed. When the option Normal is designated, there will be no picture on the screen, but it will occupy a space of the size in the font library.

**Comment by r12a** [comment]
Do i understand this correctly: you'd like a CSS property that instructed the browser to give the MVS a visible box shape, something like ▯, but only on-screen? 

Could you clarify the use cases for this?  

I'm a little worried that the hollow rectangle would look like tofu (ie. When text is rendered by a computer, sometimes characters are displayed as “tofu”. They are little boxes to indicate your device doesn’t have a font to display the text.) Is it a common practice in Mongolian?

**Comment by asmusf** [comment]
The question is whether there is a CSS property to make any other "invisible" characters visible. If not, then this shouldn't be a one-off.

**Comment by lianghai** [comment]
I agree with Asmus.

Plus, this whole requirement seems to come ultimately (the authors might have been referring to China’s national standard instead, but the standard and many other documents are derived from the Users’ Convention) from the Users’ Convention (see [MNS 4932: 2000](https://www.unicode.org/L2/topical/mongolian/MNS4932-2000(recovered).pdf), page 36 for the original text about MVS’s behavior), which was designed with an assumption of the usage of Unicode text is only about inputting with a keyboard and printing on paper. That assumption doesn’t make much sense anymore.

Merely copying the original behavior description is very much out of context of today’s various actual use cases. The whole problem of MVS’s ideal behavior needs to be reanalyzed. Husile promised to provide a kick-starter document, but he hasn’t got to it yet.

Btw, my personal preference for MVS’s behavior is, it should be displayed with a visible glyph whenever it’s not in its intended context (ie, not being used to trigger a disjointed tail of A/E). This can be resolved in fonts (although need to hack around shaping engine’s glyph-removing effort).

**Comment by r12a** [comment]
Alternatively, editor's often have a switch that allows you to see invisible characters – i have one in Dreamweaver and one in BBEdit, which are the main editors i use.  Is that sufficient?

I doubt that a CSS control is what is needed here, since CSS isn't normally used to distinguish between on-screen vs printed glyph shapes.  I'm certainly not aware that it does this for any other case.

**Comment by lianghai** [comment]
> Alternatively, editor's often have a switch that allows you to see invisible characters – i have one in Dreamweaver and one in BBEdit, which are the main editors i use. Is that sufficient?

Yeah, (programmers’ text editors are not quite relevant here but) word processors (Word, Pages, etc.) and professional typesetting applications (InDesign, etc.) do have such a switch for many general invisible characters, and the Mongolian community can benefit from this existing mechanism if they can lobby the vendors to add Mongolian’s format controls into the supported set.

Or, a font-level control may be employed. Such as a stylistic set for displaying the normally hidden format controls.
